pull.go

```diff
@@ -54,6 +54,6 @@ func pullImage(store storage.Store, options BuilderOptions, sc *types.SystemCont
 
 	logrus.Debugf("copying %q to %q", spec, name)
 
-	err = copy.Image(policyContext, destRef, srcRef, getCopyOptions())
+	err = copy.Image(policyContext, destRef, srcRef, getCopyOptions(options.ReportWriter))
```

Comment:
Ok I looked into this a little deeper, and I see that we pass different options from Buidah to Commit.  I now think we keep it the way it did.